tap/passive_tapper.go

```diff
@@ -354,7 +384,14 @@ func startPassiveTapper(outputItems chan *api.OutputChannelItem) {
 		startMemoryProfiler()
 	}
 
-	for packet := range source.Packets() {
+	for {
+		packet, err := source.NextPacket()
+		if err == io.EOF {
+			break
```

Comment:
If I understand correctly - this error should happen only when reading the packets from file and we reach the end?
so if we are reading from interfact (like the default "any") will we reach this "break"?

Reply:
@gadotroee tested with all the supported protocols and Sock Shop load test, didn't experience an `EOF`. I think `EOF` is not possible for Linux network interfaces. Do we support reading traffic from `.pcap` files? In any case, it can be turned into `continue` as a safe bet.

I've pushed more commits and updated the PR description. Please also review them.